RELEASE_NOTES.md

```diff
@@ -6,9 +6,21 @@ Chef 15 release notes will be added here as development progresses.
 
 ## Breaking Changes
 
+### Chef packages now remove /opt/chef before installation
+
+The intent of this change is that on upgrading packages the /opt/chef directory is removed of any stray `chef_gem` installed gem versions and other
```

Comment:
What does "stray" mean in this context? Also, what does "other modifications" mean?

Reply:
Consider:
The upgrade process wipes the `/opt/chef` directory before every installation and with every package manager.
You'll see a warning that the `/opt/chef` directory will be removed during the package installation process.
We've implemented this change in order to remove legacy `chef_gem` installations and any other changes to this directory that might cause issues during upgrades.
Users who are properly managing customizations to `/opt/chef` through Chef recipes won't be affected, because their customizations will still be installed by
the new chef-client package.
As to the rest of all y'alls, you should really manage your `/opt/chef` customizations with Chef recipes.
We're even here to help you.
Find us on the [Chef Community Slack](http://community-slack.chef.io/) or the [Chef Community Forum](https://discourse.chef.io/).

Reply:
Well "The upgrade process wipes the /opt/chef directory before every installation and with every package manager." is sort of covered by the "Chef packages now remove /opt/chef before installation" title.  

The block of text describing the technical details of why the change was made the way it was is necessary, since it will almost certainly cause questions in users as to why it was implemented the way it was.

I'll change the last two sentences to your text though.

Reply:
And "stray" can just be dropped, but "other modifications" means whatever anyone might have done -- "unknown unknowns"